### ISSUE TITLE
Fixed Header columns do not adjust when animating the datatable width

### ISSUE THREAD

**Issue by anthony1111** [DataTables/DataTables#826]
Hi sir, I do encountered an issue these past few days when using fixed header extendsion in datatable. The column's width are automatically adjust when resizing the browser but in my case I do have a different situation. In my case, the container of the datatable is being animated specifically the width so for example when the sidebar pops out at the left side the content will consumes the remaining space in the screen so the datatable width is animated to a smaller one but the header cell's width is not the same as the content cell's width, It will be adjusted only after when I clicked the header cell for sorting. Pls help me :(

**Comment by DataTables** [comment]
Hi - my apologies if the information in the "New Issue" template text and "Contributing" document aren't clear enough (any suggestions on how to make them more clear are welcome).

Please post support questions in the [DataTables forums](//datatables.net/forums) with a link to a test page showing the issue.
